Cargo.lock

```diff
@@ -115,9 +294,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.58"
+version = "1.0.66"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "bb07d2053ccdbe10e2af2995a2f116c1330396493dc1269f6a91d0ae82e19704"
```

Comment:
Could you please stop changing the version of the existing packages.
We can discuss if it is really necessary.

Reply:
I use `cargo add` to add dependencies. It changes the lock file automatically. Didn't notice this. Will revert it back

Reply:
Fixed 14072602bcb99dba40ed22abbd47da10676d0505